.github/workflows/goreleaser.yml

```
@@ -47,7 +48,9 @@ jobs:
           cache: false
 
       - name: Install Cosign
-        uses: sigstore/cosign-installer@e1523de7571e31dbe865fd2e80c5c7c23ae71eb4 # v3.4.0
+        uses: sigstore/cosign-installer@59acb6260d9c0ba8f4a2f9d9b48431a222b68e20 # v3.5.0
+        with:
+          cosign-release: "v2.2.4" # optional
```

Mind removing the quotes and comment?

```suggestion
          cosign-release: v2.2.4
```

Unless this accepts SHA references;  if so, my suggestion would instead be:

```yaml
          cosign-release: fb651b4ddd8176bd81756fca2d988dd8611f514d # v2.2.4
```